### PR TITLE
Add a vim repl that supports prolog

### PR DESCRIPTION
**Pull request by diogogithub**
**By submitting this pull request, I promise I have read the [contribution guidelines](https://github.com/sindresorhus/awesome-nodejs/blob/master/contributing.md) twice and ensured my submission follows it. I realize not doing so wastes the maintainers' time that they could have spent making the world better. 🖖**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -161,6 +161,7 @@
 ## Text Editor Extensions
 
 - [prolog.vim](https://github.com/adimit/prolog.vim) - Prolog integration for Vim.
+- [repl.vim](https://github.com/ujihisa/repl.vim) - VIM REPL with support for SWIProlog.
```
Comment (klaussinani) [inline readme.md]: From the [contributing guidelines](https://github.com/sindresorhus/awesome-nodejs/blob/master/contributing.md):

> - Additions should be added to the bottom of the relevant category.

Comment (diogogithub) [reply readme.md]: Ahm, sorry but, this isn't the relevant category? ._. repl.vim is an extension for the VIM text editor...

**Comment by diogogithub** [comment]
repl.vim is an extension for the VIM text editor... What would be a better category for it?

**Comment by klaussinani** [comment]
@diogogithub The current category is fine, the addition should just be placed at the bottom of it, for example:

```markdown
- [prolog.vim](https://github.com/adimit/prolog.vim) - Prolog integration for Vim.
- [ediprolog](https://github.com/triska/ediprolog) - Prolog integration for Emacs.
- [sublimeprolog](https://github.com/alnkpa/sublimeprolog) - Prolog integration
  for Sublime Text.
- [repl.vim](https://github.com/ujihisa/repl.vim) - VIM REPL with support for SWIProlog.
```

**Comment by diogogithub** [comment]
@klaussinani Got it, just pushed a fix, I misread before 😅, thanks!

**Pull request merged**